### PR TITLE
Make it clear that the commits on master were not released

### PR DESCRIPTION
**Pull request by FagnerMartinsBrack**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "jquery.cookie",
-  "version": "1.4.1",
```
Comment (FagnerMartinsBrack) [inline bower.json:3]: This is moot anyway, see https://github.com/bower/spec/commit/a325da3 and https://github.com/carhartl/jquery-cookie/pull/365

**Comment by carhartl** [comment]
The way it was done before was that notable changes were added to the "Head" section in the change log. I haven't checked, but if there are such changes we should add them as well.

**Comment by carhartl** [comment]
I still think though that this whole exercise is pointless.

**Comment by FagnerMartinsBrack** [comment]
The CHANGELOG.md is not viewed by a considerable amount of devs according to the Traffic Page. #380 complained specifically about the `*.json` files, so there is no evidence that implementing that elsewhere will fix the problem.

We can treat this as an edge case and wait to see if somebody else will report the same problem, then we will know this is relevant.

@carhartl What do you think?

**Comment by FagnerMartinsBrack** [comment]
Closing as form https://github.com/carhartl/jquery-cookie/issues/380#issuecomment-211014830

**Pull request closed without merging**